fastlane/lib/fastlane/setup/crashlytics_beta.rb

```diff
@@ -55,7 +55,7 @@ def lane_template
 
 # rubocop:disable Style/IndentationConsistency
 %{  #
-  # Learn more here: https://github.com/fastlane/setups/blob/master/samples-ios/distribute-beta-build.md 🚀
+  # Learn more here: https://github.com/fastlane/examples 🚀
```

Comment:
Shouldn't this be https://docs.fastlane.tools/getting-started/ios/beta-deployment/?